src/sqlfluff/core/config.py

```diff
@@ -15,7 +16,14 @@
 
 import appdirs
 
-import toml
+if sys.version_info >= (3, 11):
+    import tomllib
+
+    tomllib_filemode = "rb"
```

Comment:
Hmm actually I can just use `.loads` instead of `.load` and that way we don't need this binary flag, just retesting locally then will push